### DISCUSSION TITLE
#75 : Can't reach 100% 😭 I can't find why

### DISCUSSION THREAD

**Discussion by twogrey** [cssbattle/forum#34] [General]
I give up. 
Please, can someone tell me where the problem is ? 😭😭😭

![image](https://user-images.githubusercontent.com/6780374/112471058-b9040e80-8d6b-11eb-9364-e705d523c3f9.png)

**Comment by iamvdo** [comment]
Same for me. More than 80 attempts to get 100% first time. It seems that the most important part is at sharp edge between two reds. Here: 
![Screenshot](https://user-images.githubusercontent.com/1260529/112482602-86acde00-8d78-11eb-802c-1556e7c8af53.jpg)

I need to use decimal values to get good result. Good luck! :)

**Reply by SelenIT** [reply]
I agree that areas where curves meet at sharp angle are the "usual suspects" when seemingly perfect shape doesn't get 100% match. But in this particular case, I found my solutions working in a surprisingly wide range of `border-radius` values for this dark red area, even though there were visible differences in shape (however, the same shape didn't work when I tried to change approach:).

**Comment by twogrey** [comment]
I just reached the 100%. The issue was the glitched background on the nostrils.

Thx Vincent

**Comment by M4N0JKUM4R** [comment]
I have a similar problem on another one: 
![image](https://user-images.githubusercontent.com/11471878/114665705-fcbeb800-9d1a-11eb-8fa1-69432fae8b9d.png)

I can't find the glitch. 100% had worked on other problems where there were some pixels clearly misaligned to the naked eye. Anyone got an idea?

**Reply by iamvdo** [reply]
Hello. Seems like a "problem" on rendering gradient on server. Use 46.9% instead of 47%. Also, you can add two color-stops on each colors, and use 0 to make hard stops:
```css
background: linear-gradient(to right,#191919 16.5%,#eff33c 0 23.5%,#191919 0 40%,#eff33c 0 46.9%,#191919 0 63.5%,#eff33c 0)
```

**Reply by M4N0JKUM4R** [reply]
Thanks @iamvdo Just 46.9 did the trick. Is it possible to use any JS obfuscation script so that it lowers the character count. For example, I was trying to replace some big words like `background` to `bg` using the Pro feature - plugins addon and "auto run before submitting" but the server only responds to the submitted code and not the editor code as the reference. and how about the zero-width characters like this: https://www.tryhackme.com/room/theimpossiblechallenge ?
